### PR TITLE
Cop Cleanup

### PR DESCRIPTION
**Pull request by HoyaBoya**
Addreses @prsimp feedback about broken cops on this build:

https://circleci.com/gh/blueapron/dual_burner/1

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyle, SupportedStyles.
+# SupportedStyles: when_needed, always
+Style/FrozenStringLiteralComment:
+  Exclude:
+    - 'lib/dual_burner/version.rb'
```
Comment (prsimp) [inline .rubocop_todo.yml]: No need for this todo. Let's just disable this cop in the main config file.

Comment (HoyaBoya) [inline .rubocop_todo.yml]: :+1:, was taking the work around from WMS client as @TheDudeWithTheThing showed me.

Comment (HoyaBoya) [inline .rubocop_todo.yml]: I've removed the TODO and disabled the rule per your suggestion.

**Comment by HoyaBoya** [comment]
@prsimp thanks for all feedback. Hopefully all of the PoPo are happy :cop:.

File: .rubocop.yml [hunk 2]
```diff
@@ -10,10 +10,10 @@ Metrics/AbcSize:
   Max: 50 
 
 Metrics/LineLength:
-  Max: 120
+  Max: 140
```
Comment (prsimp) [inline .rubocop.yml]: 120 is somewhat a de facto length for this. The one instance that violates the cop should probably just be fixed, rather than adjusting this max to suit it. I'll comment on the offending line.

File: .rubocop.yml [hunk 3]
```diff
@@ -23,3 +23,6 @@ Style/Documentation:
 
 Style/SignalException:
   Enabled: false
+
+Style/FrozenStringLiteralComment:
```
Comment (prsimp) [inline .rubocop.yml]: We try and keep these alphabetized.

File: .rubocop.yml (hunk 2 above)
Comment (HoyaBoya) [inline .rubocop.yml]: The offending line was due to a Thor DSL declaration. I can re-impose 120, then work around Thor.

File: lib/dual_burner/runner.rb [hunk 4]
```diff
@@ -22,10 +25,14 @@ def last_release
     method_option :config, aliases: '-c', type: :string, optional: true, desc: CONFIG_DESC
     method_option :git_reference, aliases: '-g', default: 'master', type: :string, required: false, desc: 'The git reference to deploy.'
```
Comment (prsimp) [inline lib/dual_burner/runner.rb]: This is the line that is too long (136/120). A couple notes:

* above, we use `required: true` and here we use `required: false`. Is one of these not the default value?
* perhaps we could use "ref" instead of "reference", which IMO feels more "git-y". If not, perhaps "sha" is a good choice.
* If we can't shorten this to 120, IMO it should just break. I realize it's a bit less visually appealing, but I do think it's good for us to stick to the 120 limit (especially given many folks would argue for setting it to 80!).

**Comment by HoyaBoya** [comment]
@prsimp fixed the 120 vs 140 and addressed the offending Thor line.

File: lib/dual_burner/runner.rb (hunk 4 above)
Comment (HoyaBoya) [inline lib/dual_burner/runner.rb]: I just made a constant `GIT_DESC` for now.

Comment (HoyaBoya) [inline lib/dual_burner/runner.rb]: It's not really a SHA...and there is a `-g` shortcut. This could be `optional: true` to be more consistent with the Thor usage.

**Comment by prsimp** [comment]
Looks good, thanks for cleaning this up! :+1:

**Comment by HoyaBoya** [comment]
Thanks for keeping me on the right side of the road chief!
